cogs/config.py

```diff
@@ -3,7 +3,7 @@
 IMAGE_PATH = 'image.png'
 IMAGE_RESIZED = 'image-resized.png'
 IMAGE_GRAY = 'image-gray.png'
-IMAGE_FINAL = 'image-final.png'
+IMAGE_THRESH = 'image-thresh.png'
```

Comment:
final makes sense chronologically, but not isolated. thresh is more descriptive in this regard.